src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,8 +14,10 @@
     private Phone phone;
     private Email email;
     private Address address;
+    public int sequenceNumber;
 
     private final UniqueTagList tags;
+    public static int nextSequenceNumber = 0;
     /**
```

Comment:
It will work. However, it more meaningful to init to 1 since the variable is called next SequenceNumber and post increment.